auth-web/src/components/CardHeader.vue

```diff
@@ -0,0 +1,64 @@
+<template>
```

Comment:
The Card header:
![card header](https://github.com/bcgov/sbc-auth/assets/122301442/d370f472-4cb6-4aec-832f-7434fde12bae)